erdjs/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes will be documented in this file.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [2.2.1] - 10.02.2021
```

Comment:
I would recommend using the `yyyy.mm.dd` format for the future releases. What do you think @ccorcoveanu @danielailie ?